backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/database/repository/NormManifestationRepository.java

```diff
@@ -96,4 +99,10 @@ void updatePublishStateByManifestationEli(
     @Param("manifestationEli") String manifestationEli,
     @Param("publishState") NormPublishState publishState
   );
+
+  @NativeQuery(
+    value = "select DISTINCT ON (eli_norm_work) * from norm_manifestation ORDER BY eli_norm_work, eli_norm_manifestation DESC",
```

Comment:
Why do we need to also order DESC by eli_norm_manifestation?

Reply:
We need to sort the expressions of a work descendingly so we load the title from the newest expression (and then also the newest manifestation of that expression) Thats what `eli_norm_manifestation DESC` takes care of. The ascending order of the works is taken care of by `ORDER BY eli_norm_work` as that defaults to `ASC`.
I add in the `ASC` explicitly so it is clearer what is happening here.

Reply:
I guess you'll be reworking how we retrieve the data in your new ticket but the current SQL does not seem correct to me. Since the `ORDER BY eli_norm_work` is in the inner SQL and yes it defaults to ASC but since it is in the inner SQL doing the distinct, the order by only determines which row to pick, but does not guarantee the final result is ordered by eli_norm_work. For that we would need to do the order in the outer SQL

Reply:
oh your right it's not guaranteed to work (but does accidentally), I added a second order by into the outer query on the version i created in the new ticket